algebird-core/src/main/scala/com/twitter/algebird/BloomFilter.scala

```diff
@@ -20,49 +20,156 @@ import scala.collection.immutable.BitSet
 import scala.collection.JavaConverters._
 
 import com.googlecode.javaewah.{ EWAHCompressedBitmap => CBitSet }
+import com.googlecode.javaewah.IntIterator
 
 object RichCBitSet {
-  def apply(x: Int*) = {
-    CBitSet.bitmapOf(x.sorted: _*)
+  def apply(xs: Int*): CBitSet = fromArray(xs.toArray)
+
+  // this sorts the array in-place
+  def fromArray(x: Array[Int]): CBitSet = {
+    val bs = new CBitSet
```

Comment:
not the examples I see from their code. It is doing compression of the differences between the indices. I don't see immediately how such a hit would be used.